sequencer.c

```diff
@@ -1013,10 +1025,17 @@ static int run_git_commit(struct repository *r,
 
 		if (res <= 0)
 			res = error_errno(_("could not read '%s'"), defmsg);
-		else
+		else {
+			if (opts->ignore_date) {
+				if (!author)
+					BUG("rebase -i must set author "
+					    "before committing tree");
```

Comment:
Maybe expand this to something like "ignore_date can only be used with rebase -i, which must set the author befor committing the tree"?